### ISSUE TITLE
simpletest is flakey in the testing of citadel

### ISSUE THREAD

**Issue by sdake** [istio/istio#11358]
**Describe the bug**
A PR that removes silent failures from the simpletest check results in a hard failure on a specific test case.  The test case fails to readiness check citadel.

**Expected behavior**
The test case should work appropriately.

**Steps to reproduce the bug**
submit a PR like :
**Version**
master/releasee-1.1

**Installation**
none

**Environment**
Prow simpleTest check

**Cluster state**
```
FAILED: istio-security-test, run `kubectl logs istio-security-test --namespace istio-system` for more info
As in https://storage.googleapis.com/istio-prow/logs/e2e-simpleTests/16580/build-log.txt
```

**Comment by sdake** [comment]
master PR:  https://github.com/istio/istio/pull/11359
release-1.1 PR cherrypick: https://github.com/istio/istio/pull/11360

**Comment by sdake** [comment]
@hklai worked with me this evening to identify this as a problem for 1.1.  The goal of these two related but different problems is to remove failures in the gate for master and release-1.1 to speed up development.

**Comment by sdake** [comment]
This issue can be closed as the master and release-1.1 cherrypick have merged.
